ci/cray/Jenkinsfile.verbs

```diff
@@ -0,0 +1,147 @@
+// Libfabric Verbs provider Jenkins Pipeline file
+// Copyright 2018. Cray Inc.
+
+
+pipeline {
+    options {
+        // Generic build options
+        timeout (time: 30, unit: 'MINUTES')
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+
+        // Build options
+        disableConcurrentBuilds()
+        skipStagesAfterUnstable()
+        timestamps()
+    }
+   agent {
+        node {
+            label 'wham'
+        }
+    }
+    stages {
+        stage('Build') {
+            steps {
+                sh './autogen.sh'
+                sh './configure --prefix=$TMP_INSTALL_PATH/'
+                sh 'make -j 12'
+                sh 'make install'
+            }
+        }
+        stage('Test: Phase 1') {
+            failFast true
+            parallel {
+                stage('Test-Unit') {
+                    steps {
+                        echo 'stubbed-unit'
+                        sh 'true'
+                    }
+                }
+                stage('Test-Smoke') {
+                    steps {
+                        echo 'stubbed-smoke'
+                        sh 'true'
+                    }
+                }
+                stage('Test-Functional') {
+                    steps {
+                        echo 'stubbed functional'
+                        sh 'true'
+                    }
+                }
+                stage('Test-Fabtests') {
+                    steps {
+                        timeout (time: 10, unit: 'MINUTES') {
+                            withEnv(['FI_PROVIDER=\'ofi-rxm;verbs\'', 'LD_LIBRARY_PATH="$TMP_INSTALL_DIR/lib:$LD_LIBRARY_PATH"']) {
+                                sh 'srun -p mellanox -n 1 -N 1 -w wham-0-cn8 $FABTEST_PATH/bin/runfabtests.sh -p $FABTEST_PATH/bin -v -T 60 \'verbs\' 10.100.49.8 10.100.49.9 || true'
```

Comment:
Not certain this is relevant, but I have to specify "verbs;ofi_rxm" to runfabtests.sh to select the correct provider, otherwise I just get the the verbs provider.